source/includes/wp-api-v3/_taxes.md

```diff
@@ -9,8 +9,10 @@ The taxes API allows you to create, view, update, and delete individual tax rate
 | `id`       | integer | Unique identifier for the resource. <i class="label label-info">read-only</i>                                                                                                   |
 | `country`  | string  | Country ISO 3166 code. See [ISO 3166 Codes (Countries)](http://www.chemie.fu-berlin.de/diverse/doc/ISO_3166.html) for more details                                              |
 | `state`    | string  | State code.                                                                                                                                                                     |
-| `postcode` | string  | Postcode/ZIP.                                                                                                                                                                   |
-| `city`     | string  | City name.                                                                                                                                                                      |
+| `postcode` | string  | Postcode/ZIP. Deprecated (doesn't support multiple values), `postcodes` should be used instead.                                                                                 |
+| `city`     | string  | City name. Deprecated (doesn't support multiple values), `cities` should be used instead.                                                                                       |
+| `postcodes`| string  | Postcodes/ZIPs.                                                                                                                                                                 |
```

Comment:
I think is better to indicate in what version of WooCommerce those got included. And also when we deprecated, since it will affect REST API calls and webhooks as well, so people need to be aware when those changes started.